README.md

```diff
@@ -1,38 +1,36 @@
-# The Swift Community Podcast
+<p align=center><img src="https://github.com/SwiftCommunityPodcast/podcast/blob/master/Resources/Logo%402x.png" width="250"/></p>
 
-## About
+### 🎙 For the community. By the community.
 
-<img src="https://github.com/SwiftCommunityPodcast/podcast/blob/master/Resources/Logo%402x.png" width="250"/>
+Welcome, we're glad you're here! The Swift Community Podcast is a community-driven show focusing on the people that build, use, and support Swift, Apple's open source programming language. This repository is a place where anyone who is interested can stay informed about the latest developments, learn new techniques and engage with others in the community.
 
-A community driven show focusing on the people that build, use, and support Swift, Apple's open source programming language.
+### 🤝 Contributing 
 
-Everyone is welcome to contribute as a producer, host, guest, editor, publisher or by suggesting show topics or anything else you can think of!
+Swift is a language for everyone...from the earliest beginner to most hardcore and everyone in between. No contribution is too small!
```

Comment:
This makes it seem like this repo is about contributing to the Swift project. Consider changing the string to something like: `Swift is a language for everyone...from beginners to experienced programmers and everyone in between, so we welcome contributions to the podcast from people of any experience level. No contribution is too small!`